client/dashboard/profile-wizard/steps/industry.js

```diff
@@ -85,9 +91,20 @@ class Industry extends Component {
 		);
 	}
 
+	onInputChange( value, slug ) {
```

Comment:
Would `onDetailChange` be a more descriptive/accurate name?

Reply:
You are right, I'm changing the name now.